src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+    add(a) {
+      calculator.result += a;
```

Comment:
```suggestion
      this.result += a;
```
Fix all cases

Reply:
if i put this instead of calculator i get errors  . before was doing with this and didnt figure out why its not working so had to put calculator

Reply:
i had to find how call() works and call() js function in general, or i missed video lecture where it was mentioned ?

Reply:
You could also pass this.result as an argument of operate and add a value to it. 
Or save the result of operate function into this.result. 
I guess you will have theory and videos about call, bind and apply in future sections, but here is some additional theory for you, if you want to know more :)
https://www.freecodecamp.org/news/understand-call-apply-and-bind-in-javascript-with-examples/